client/components/App.jsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react'
-import { BrowserRouter as Router, Switch } from 'react-router-dom'
+
+import Login from './Login'
+
+import { BrowserRouter as Router, Switch, Route  } from 'react-router-dom'
+
 
 const App = () => {
   return (
     <Router>
+
       <Switch>
-        <h1>MOA development has begun!</h1>
+        <Route path='/' component ={Login} />
```

Comment:
You may need to remove what you've done here, and merge the current dev branch as that has the client side routing setup.

Reply:
Code changed. thanks, please review